HLTriggerOffline/Btag/src/HLTBTagHarvestingAnalyzer.cc

```diff
@@ -46,13 +46,29 @@ HLTBTagHarvestingAnalyzer::dqmEndJob(DQMStore::IBooker & ibooker, DQMStore::IGet
 				efficsOK[flavour]=isOK;
 			}
 			label= m_histoName.at(ind)+std::string("___");
+			TString labelEta = label;
```

Comment:
Hi @natalia-korneeva - as not to make the existing issues with string conversions in this class worse, please change the types of labelEta and labelPhi to be std::string. That will avoid several conversions between string<->Tstring in the code you've added below.